crates/lox-io/src/ndm/common.rs

```diff
@@ -0,0 +1,1624 @@
+    #[serde(rename = "CREATION_DATE")]
+    pub creation_date: EpochType,
+    #[serde(rename = "ORIGINATOR")]
+    pub originator: String,
+}
+
+#[derive(Clone, Debug, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+#[serde(default)]
+pub struct AdmHeader {
+    #[serde(rename = "COMMENT")]
+    pub comment_list: Vec<String>,
+    #[serde(rename = "CREATION_DATE")]
+    pub creation_date: EpochType,
+    #[serde(rename = "ORIGINATOR")]
+    pub originator: String,
+    #[serde(rename = "MESSAGE_ID")]
+    pub message_id: Option<String>,
+}
+
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    lox_derive::KvnDeserialize,
+)]
+#[serde(default)]
+pub struct OdmHeader {
+    #[serde(rename = "COMMENT")]
+    pub comment_list: Vec<String>,
+    #[serde(rename = "CLASSIFICATION")]
+    pub classification_list: Vec<String>,
+    #[serde(rename = "CREATION_DATE")]
+    pub creation_date: EpochType,
+    #[serde(rename = "ORIGINATOR")]
+    pub originator: String,
+    #[serde(rename = "MESSAGE_ID")]
+    pub message_id: Option<String>,
+}
+
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    lox_derive::KvnDeserialize,
+)]
+#[serde(default)]
+#[kvn(value_unit_struct)]
+pub struct AccType {
+    #[serde(rename = "$text")]
+    pub base: f64,
+    #[serde(rename = "@units")]
+    pub units: Option<AccUnits>,
+}
+
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    lox_derive::KvnDeserialize,
+)]
+#[serde(default)]
+#[kvn(value_unit_struct)]
+pub struct AngleType {
+    #[serde(rename = "$text")]
+    pub base: f64,
+    #[serde(rename = "@units")]
+    pub units: Option<AngleUnits>,
+}
+
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    lox_derive::KvnDeserialize,
+)]
+#[serde(default)]
+#[kvn(value_unit_struct)]
+pub struct AngleRateType {
+    #[serde(rename = "$text")]
+    pub base: f64,
+    #[serde(rename = "@units")]
+    pub units: Option<AngleRateUnits>,
+}
+
+#[derive(Clone, Debug, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+#[serde(default)]
+pub struct AngMomentumType {
+    #[serde(rename = "$text")]
+    pub base: f64,
+    #[serde(rename = "@units")]
+    pub units: AngMomentumUnits,
+}
+
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    lox_derive::KvnDeserialize,
+)]
+#[serde(default)]
+#[kvn(value_unit_struct)]
+pub struct AngVelComponentType {
+    #[serde(rename = "$text")]
+    pub base: f64,
+    #[serde(rename = "@units")]
+    pub units: Option<AngleRateUnits>,
+}
+
+#[derive(Clone, Debug, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+#[serde(default)]
+pub struct AngVelStateType {
+    #[serde(rename = "COMMENT")]
+    pub comment_list: Vec<String>,
```

Comment:
The `_list` is on purpose. This is all generated code, so I did not spend much time on this.

Since you explained this is minor, I'll mark this as resolved.